roles/common/templates/etc/serverspec/os_spec.rb

```diff
@@ -0,0 +1,149 @@
+require 'spec_helper'
+
+describe file('/etc/pam.d/login') do
+    it {should contain '^@include common-auth' }
+    it {should contain '^@include common-account' }
+    it {should contain '^@include common-session' }
+    it {should contain '^@include common-password' }
+end
+
+describe file('/etc/pam.d/common-password') do
+  it { should contain '^password [success=7 default=ignore] pam_unix.so obscure sha512' }
+end
+
+describe file('/etc/login.defs') do
+  it { should contain 'PASS_MAX_DAYS   90' }
+  it { should contain 'PASS_MIN_DAYS   1' }
```

Comment:
Lets just go with checking the login.defs. It'll be faster than enumerating through each user on the system.